content/doc/developer/security/csp.adoc

```diff
@@ -95,6 +99,24 @@ You can generally use https://github.com/jenkinsci/stapler/blob/master/docs/jell
 
 An example of this is https://github.com/jenkinsci/jenkins/pull/6849[jenkinsci/jenkins#6849].
 
+==== Inline `st:bind` tags
+
+If you're using `<st:bind>` tags inside inline JavaScript, add the `var` attribute to set the variable with the specified name, and reference that variable in your own scripts.
+Use a simple JavaScript identifier (complying with https://github.com/jenkinsci/stapler/blob/92458dd7afca3061956ccf92fedf1782a6e76e39/core/src/main/java/org/kohsuke/stapler/bind/BoundObjectTable.java#L59-L63[this regular expression]) as `var` value, rather than a more complex expression.
+
+Make sure to not reuse the same `var` value when extracting multiple or repeatedly executed `<st:bind/>` tags from inline JS.
```

Comment:
```suggestion
Make sure to not reuse the same `var` value when extracting multiple or repeatedly executed `<st:bind>` tags from inline JS.
```
Or is this intentional?

Reply:
Kind of an indicator for a tag with no content. Doesn't really matter, either works I guess.

Reply:
That slash really stands out to me for some reason, but I can live with it.